PrestaShop/distribution-api
# Put assets (modules, logo, PrestaShop versions) in the bucket

| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | The main purpose of this PR is to upload module zips and logo & PrestaShop zips to the bucket and make the API returns the url of those files instead of using Github files. It also add some optimizations and add `download_url` & `stability` information on the `/prestashop` & `/prestashop/{stable,rc,beta}` endpoints.
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #3 #10 & #11 
| How to test?      |  See below
| Possible impacts? | 

### How to test
1. Replace the url in `modules/ps_distributionapiclient/src/DistributionApi.php` line 36 by:
```diff
-private const API_ENDPOINT = 'https://api.prestashop-project.org';
+private const API_ENDPOINT = 'https://integration-api.prestashop-project.org';
```
2. Clear cache: `rm -rf var/cache/*`
3. Fully uninstall any native modules (psgdpr for instance)
4. You should be able to install it back when `ps_distributionapiclient` is enable
